src/Models/AbstractWidget.php

```diff
@@ -89,13 +87,11 @@ public function widget( $args, $instance ) {
 	 * @param array $instance Previously saved values from database.
 	 */
 	public function form( $instance ) {
-		$title = ! empty( $instance['title'] )
-			? $instance['title']
-			: '';
+		$title = empty( $instance['title'] ) ? '' : $instance['title'];
 		?>
 		<p>
 			<label for="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>">
-				<?php esc_html_e( 'Title:' ); ?>
+				<?php esc_html( 'Title:' ); ?>
```

Comment:
The domain is actually optional. But anyway, if you want to change it, it needs to echo the result, what you have now is not the same.